recipe_scrapers/__init__.py

```diff
@@ -338,6 +339,8 @@
     BakingSense.host(): BakingSense,
     BakingMischief.host(): BakingMischief,
     BareFootContessa.host(): BareFootContessa,
+    Bergamot.host(): Bergamot,
+    Bergamot.host(): Bergamot,
```

Comment:
Fixup: duplicate entries here:

```suggestion
    Bergamot.host(): Bergamot,
```